scripts/install.sh

```diff
@@ -0,0 +1,145 @@
+#!/bin/sh
+
+# Set shell options based on shell type
+if [ -n "$BASH_VERSION" ]; then
+    set -euo pipefail
+else
+    set -eu
+fi
+
+# Default log level and application details
+LOG_LEVEL=${LOG_LEVEL:-INFO}
+APP_NAME=${APP_NAME:-"wazuh-agent-status"}
+WOPS_VERSION=${WOPS_VERSION:-"0.2.1"}
```

Comment:
Why the need of this variable ?

Reply:
This file isn't the point of interest for now.